index.bs

```diff
@@ -176,19 +185,101 @@ When an item is of type <dfn value export for="AV1 Image Item Type">av01</dfn>,
+
+<h4 id ="layered-properties">Properties</h4>
+<h5 id ="operating-point-selector-property">Operating Point Selector Property</h5>
+
+<h6 id ="operating-point-selector-property-definition" class="no-toc">Definition</h6>
+
+<pre class="def">
+  Box Type:       <dfn value export for="OperatingPointSelectorProperty">a1op</dfn>
+  Property type:  Descriptive item property
+  Container:      <code>ItemPropertyContainerBox</code>
+  Mandatory:      No
+  Quantity:       One
+</pre>
+
+<h6 id="operating-point-selector-property-description" class="no-toc">Description</h6>
+
+An <dfn>OperatingPointSelectorProperty</dfn> may be associated with an [=AV1 Image Item=] to provide the index of the operating point to be processed for this item. If associated, it shall be marked as essential.
+
+In the absence of a 'lsel' property associated with the item:
+- If an [=OperatingPointSelectorProperty=] is associated with an [=AV1 Image Item=], the 'ispe' property associated with this item shall describe the dimension of the output image corresponding to the operating point.
+- If it is not associated with an item, the 'ispe' documents the dimensions of the the operating point whose index is 0.
+
+<h6 id ="operating-point-selector-property-syntax" class="no-toc">Syntax</h6>
+
+```
+class OperatingPointSelectorProperty extends ItemFullProperty('a1op', version = 0, flags) {
+    u8 op_index;
+}
+```
+
+<h6 id ="operating-point-selector-property-semantics" class="no-toc">Semantics</h6>
+
+<dfn noexport>op_index</dfn> indicates index of the operating point to be processed for this item. Its value shall be between 0 and [=operating_points_cnt_minus_1=].
+
+<h5 id ="layer-selector-property">Layer Selector Property</h5>
+The 'lsel' property defined in [[!HEIF]] may be associated with an [=AV1 Image Item=]. The 'layer_id' indicates the value of the [=spatial_id=] to render. The value shall be between 0 and 3. The corresponding spatial layer shall be present in the bitstream and shall produce an output frame. Other layers may be needed to decode the indicated layer.
+
+If this property is associated with an item, the 'ispe' property documents the dimensions of the output frame produced by decoding this layer.
+
+NOTE: The dimensions of possible intermediate output images might not match the ones given in the 'ispe' property. Renderers are expected to scale the output image to match the 'ispe' property.
```

Comment:
This NOTE seems to be in the wrong place. If the 'lsel' property is present, the layers not selected by the 'ispe' property will not be rendered. The renderer should not need to scale the output images of the unselected layers.

It seems that this NOTE should be moved to the previous subsection, at line 234, after the paragraph "In the absence of a 'lsel' property associated with the item".

Reply:
Even if you select a layer, you may have to decode intermediate images whose dimensions might differ but you're right that in this case the expectation is that you don't render these intermediate images so scaling should not be needed. I moved the note as suggested.